x/ccv/provider/migrations/v6/migrations.go

```diff
@@ -0,0 +1,33 @@
+package v6
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	providerkeeper "github.com/cosmos/interchain-security/v4/x/ccv/provider/keeper"
+)
+
+func MigrateMinPowerInTopN(ctx sdk.Context, providerKeeper providerkeeper.Keeper) {
+	// we only get the registered consumer chains and not also the proposed consumer chains because
+	// the minimal power is usually set when the consumer chain addition proposal passes
```

Comment:
It is _always_, right? 
```suggestion
	// the mainimal power is set when the consumer chain addition proposal passes
```